Week1/homework/world-queries.txt

```diff
@@ -37,4 +37,180 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
++----------------------------------+
+| Aruba                            |
+| Anguilla                         |
+| Netherlands Antilles             |
+| Antigua and Barbuda              |
+| Bahamas                          |
+| Belize                           |
+| Bermuda                          |
+| Barbados                         |
+| Canada                           |
+| Costa Rica                       |
+| Cuba                             |
+| Cayman Islands                   |
+| Dominica                         |
+| Dominican Republic               |
+| Guadeloupe                       |
+| Grenada                          |
+| Greenland                        |
+| Guatemala                        |
+| Honduras                         |
+| Haiti                            |
+| Jamaica                          |
+| Saint Kitts and Nevis            |
+| Saint Lucia                      |
+| Mexico                           |
+| Montserrat                       |
+| Martinique                       |
+| Nicaragua                        |
+| Panama                           |
+| Puerto Rico                      |
+| El Salvador                      |
+| Saint Pierre and Miquelon        |
+| Turks and Caicos Islands         |
+| Trinidad and Tobago              |
+| United States                    |
+| Saint Vincent and the Grenadines |
+| Virgin Islands, British          |
+| Virgin Islands, U.S.             |
++----------------------------------+
+4. What's the top 10 countries by SurfaceArea ?
+mysql> SELECT name
+    -> FROm country ORDER by surfacearea DESC LIMIT 10;
++--------------------+
+| name               |
++--------------------+
+| Russian Federation |
+| Antarctica         |
+| Canada             |
+| China              |
+| United States      |
+| Brazil             |
+| Australia          |
+| India              |
+| Argentina          |
+| Kazakstan          |
++--------------------+
+5. What's the top 10 most populated cities?
+mysql> SELECT name
+    -> FROM city ORDER BY population DESC LIMIT 10;
++-------------------+
+| name              |
++-------------------+
+| Mumbai (Bombay)   |
+| Seoul             |
+| São Paulo         |
+| Shanghai          |
+| Jakarta           |
+| Karachi           |
+| Istanbul          |
+| Ciudad de México  |
+| Moscow            |
+| New York          |
++-------------------+
+6. Select the top 3 countries by population that their names start with the letter ‘P’
+mysql> SELECT name
+    -> FROM country
+    -> WHERE name like "p%" ORDER BY population LIMIT 3;
++----------+
+| name     |
++----------+
+| Pitcairn |
+| Palau    |
+| Panama   |
++----------+
+
+7. What is the population in Asia?
+mysql> SELECT sum(population)
+    -> FROM country
+    -> WHERE continent="ASIA";
++-----------------+
+| sum(population) |
++-----------------+
+|      3705025700 |
++-----------------+
+8. Which languages are spoken in the Region of ‘South America’
+mysql> SELECT language
```

Comment:
You can use `SELECT DISTINCT language` here so that you don't see the same language more than once in the results (for example Spanish and Italian are repeated more than once in the results).